datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/analytics/resolver/GetChartsResolver.java

```diff
@@ -88,19 +89,25 @@ private TimeSeriesChart getActiveUsersTimeSeriesChart(
    */
   private List<AnalyticsChart> getProductAnalyticsCharts(Authentication authentication) throws Exception {
     final List<AnalyticsChart> charts = new ArrayList<>();
-    final DateTime now = DateTime.now();
-    final DateTime aWeekAgo = now.minusWeeks(1);
+    final DateTime endOfWeek = DateTime.now()
+            .plusDays(1)
```

Comment:
I assume you add  `plusDays(1)` here in case today is Sunday and we need to get the following sunday? and this is all good if today is Saturday?

Reply:
What I wanted was the exact end of Sunday. I initially tried that by setting things as 23:59:59.999 but the rendering on the frontend added one point with zero for the exact end. So decided to make it the exact beginning of next Monday by adding plus 1 days.